mobile-app/app/screens/AppNavigator/screens/Loans/components/AddOrRemoveCollateralForm.tsx

```diff
@@ -114,6 +113,17 @@ export const AddOrRemoveCollateralForm = memo(({ route }: Props): JSX.Element =>
     setCollateralValue(amount)
   }
 
+  const currentBalance = vault?.collateralAmounts?.find((c) => c.id === token.id)?.amount ?? '0'
```

Comment:
Are we missing the initial scenario here? Like, if it's the only VS, we display 100% immediately. The result might be the same. I just want to save on calculation in case it's 100%.

Reply:
Meaning, we can short-circuit by not needing to calculate it in case there are no other collaterals.